core/templates/components/graph-services/graph-layout.service.ts

```diff
@@ -520,7 +520,8 @@ export class StateGraphLayoutService {
   }
 
   getAugmentedLinks(
-      nodeData: NodeDataDict, nodeLinks: GraphLink[]): AugmentedLink[] {
+      nodeData: NodeDataDict,
+      nodeLinks: GraphLink[]): AugmentedLink[] | undefined {
```

Comment:
```suggestion
      nodeData: NodeDataDict,
      nodeLinks: GraphLink[]
  ): AugmentedLink[] | undefined {
```

Why can it return `undefined`?

Reply:
I did this to remove the TS error given below:
`core/templates/components/graph-services/graph-layout.service.ts(542,11): error TS2322: Type 'undefined' is not assignable to type 'AugmentedLink[]'.`

Refer lines 541-543 of the same file, you will see we are not returning anything.
![image](https://user-images.githubusercontent.com/75239620/143454765-d59b910e-9b42-4c88-a28a-8dc311705a2b.png)